lib/virt_autotest/utils.pm

```diff
@@ -429,7 +429,17 @@ sub wait_guest_online {
     my $guest = shift;
     my $retries = shift // 300;
     # Wait until guest is reachable via ssh
-    script_retry("nmap $guest -PN -p ssh | grep open", delay => 1, retry => $retries);
+    if (script_retry("nmap $guest -PN -p ssh | grep open", delay => 1, retry => $retries, die => 0) != 0) {
+        # Ensure guest is running
+        if (script_run("virsh list --name --state-running | grep $guest") != 0) {
```

Comment:
@tonyyuan1  @tbaev  This change may impact maintenance test. Would you like to have a check if all is good with this change?